src/components/EditSections/EditUserInfo/EditUserInfo.js

```diff
@@ -282,6 +298,8 @@ class EditUserInfo extends React.Component {
                 defaultValue={initialValues.expirationDate}
                 name="expirationDate"
                 id="adduser_expirationdate"
+                timeZone={timezone}
```

Comment:
No `timeZone` needed here since date and time value is processing  in `parseExpirationDate`

Reply:
Fixed